Sources/PodToBUILD/XCConfig.swift

```diff
@@ -42,7 +42,11 @@ public struct XCConfigTransformer {
 
         let allValues = value.components(separatedBy: CharacterSet.whitespaces)
         return allValues.filter { $0 != "$(inherited)" }
-            .compactMap { transformer.string(forXCConfigValue: $0) }
+            .compactMap { val in
+                return transformer.string(forXCConfigValue: val)?
+                    .replacingOccurrences(of: "$(PODS_ROOT)", with: "Vendor")
```

Comment:
Longer term, should we consider this being user specified rather than a fixed "vendor" directory? Also how does the work with external if we use repository rules?

Reply:
Currently, `external` isn't supported and there is no notion of `external` in P2B. #36 has some notes about potentially supporting it.